exercises/queen-attack/canonical-data.json

```diff
@@ -8,119 +10,133 @@
     "the board graphically, or using standard chess notation. Those",
     "tests can be offered as extra credit"
   ],
-  "create": {
-    "description": "Test creation of Queens with valid and invalid positions",
-    "cases" : [
-      {
-        "description": "queen with a valid position",
-        "queen": {
-          "position": "(2,2)"
+  "cases": [
+    {
+      "description": "Test creation of Queens with valid and invalid positions",
+      "cases": [
+        {
+          "description": "queen with a valid position",
+          "property": "create",
+          "queen": {
+            "position": "(2,2)"
+          },
+          "expected": 0
         },
-        "expected": 0
-      },
-      {
-        "description": "queen must have positive rank",
-        "queen": {
-          "position": "(-2,2)"
+        {
+          "description": "queen must have positive rank",
+          "property": "create",
+          "queen": {
+            "position": "(-2,2)"
+          },
+          "expected": -1
```

Comment:
Shouldn't `-1` be replaced with an `error` object? If so, the description in the "comments" key should also be updated.